### ISSUE TITLE
No apparent way to launch/dock fighters

### ISSUE THREAD

**Issue by AbrarSyed** [thewierdnut/endless-mobile#21]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

There is no way to launch/dock fighters. There is no button on the UI for this. I have also tried some other stuff like touch-and-hold on my flagship or other ships, as well as double taps.

### Steps to Reproduce

1. Obtain a ship with fighter bays
2. Obtain fighters to live in those bays
3. Launch out of a planet to space
4. Observe no button to launch fighters

### Expected Behavior

A button similar to attack, switch weapons, talk, scan, etc to launch/dock fighters

### Device type

Android 12

### Game Version

0.9.16.1-28.28 F-Droid

### Additional Information

I attempted to dig into the source code and put up a PR for this. I found how to add UI elements and such, but couldn't find where the actual action to launch/dock fighters happens.

**Comment by thewierdnut** [comment]
Deploying fighters is not a keypress, its a reconfigurable ship command, which is handled slightly differently. 

Assuming you managed to add a button via the interface, you need to add a line like this to MainPanel::KeyDown()

```
	else if(key == SDLK_d) // synthetic keypress via UI button, not keyboard
		Command::Inject(Command::DEPLOY);
```

**Comment by thewierdnut** [comment]
Fixed in pull request #23
